src/Sarif/Converters/FxCopConverter.cs

```diff
@@ -580,23 +580,35 @@ public void Read(Context context, Stream input)
         {
             XmlSchemaSet schemaSet = new XmlSchemaSet();
             Assembly assembly = typeof(FxCopLogReader).Assembly;
+            Stream stream = null;
 
-            using (var stream = assembly.GetManifestResourceStream(FxCopLogReader.FxCopReportSchema))
-            using (var reader = XmlReader.Create(stream))
+            try
             {
-                XmlSchema schema = XmlSchema.Read(reader, new ValidationEventHandler(ReportError));
-                schemaSet.Add(schema);
-            }
+                stream = assembly.GetManifestResourceStream(FxCopLogReader.FxCopReportSchema);
+                using (var reader = XmlReader.Create(stream))
+                {
+                    stream = null;
```

Comment:
Yes, this is exactly the fix that the help page for this rule (https://msdn.microsoft.com/en-us/library/ms182334.aspx) suggests.